packages/react/popper/src/Popper.tsx

```diff
@@ -86,14 +84,13 @@ const Popper = React.forwardRef((props, forwardedRef) => {
   const isPlaced = placedSide !== undefined;
 
   return (
-    <div style={popperStyles} {...(selector ? getSelectorObj(selector + '-wrapper') : undefined)}>
+    <div style={popperStyles} data-radix-popper-wrapper="">
```

Comment:
inlined for now but will be replaced by a `wrapperClassName` in another PR later.